src/libs/histogram.c

```diff
@@ -1346,7 +1346,7 @@ static gboolean _lib_histogram_scroll_callback(GtkWidget *widget, GdkEventScroll
      dt_modifier_is(event->state, GDK_CONTROL_MASK) && !darktable.gui->reset)
   {
     /* set size of navigation draw area */
-    const float histheight = clamp_range_f(dt_conf_get_int("plugins/darkroom/histogram/height") * 1.0f + 10 * delta_y, 100.0f, 200.0f);
+    const float histheight = clamp_range_f(dt_conf_get_int("plugins/darkroom/histogram/height") * 1.0f + 10 * delta_y, 100.0f, 250.0f);
```

Comment:
while it might be a bit more verbose, you could use dt_confgen_get_int with DT_MIN and DT_MAX as params, which would then remove dependency on magic numbers and instead rely just on config data?

Reply:
I don't understand. This is getting the value, adjusting it, then clamping the result, not just clamping the retrieved integer. How does dt_confgen_get_int help here?

Reply:
I could get it, adjust it, set it, then get it again, which would clamp using config values. Is that what you mean?

Reply:
I mean something like this:
```suggestion
    const float histheight = clamp_range_f(dt_conf_get_int("plugins/darkroom/histogram/height") * 1.0f + 10 * delta_y, (float)dt_confgen_get_int("plugins/darkroom/histogram/height", DT_MIN), (float)dt_confgen_get_int("plugins/darkroom/histogram/height", DT_MAX));
```

Reply:
Ah I see. I misunderstood what the function was doing.

Reply:
done